### PR TITLE
Releng/update to java 14  and report richtext

### PR DESCRIPTION
**Pull request by arnauddieumegard**

### CODE REVIEW

File: richtext/features/org.polarsys.kitalpha.richtext.widget.feature/feature.xml [hunk 1]
```diff
@@ -30,11 +30,6 @@
    <includes
```
Comment (bzendagui) [inline richtext/features/org.polarsys.kitalpha.richtext.widget.feature/feature.xml:30]: Please update the copyright year in the copyright header

File: richtext/plugins/org.polarsys.kitalpha.richtext.common/src/org/polarsys/kitalpha/richtext/common/impl/AbstractMDERichTextWidget.java [hunk 2]
```diff
@@ -14,12 +14,9 @@
 import java.beans.PropertyChangeListener;
```
Comment (bzendagui) [inline richtext/plugins/org.polarsys.kitalpha.richtext.common/src/org/polarsys/kitalpha/richtext/common/impl/AbstractMDERichTextWidget.java:14]: Space and tabulation problem?

File: richtext/plugins/org.polarsys.kitalpha.richtext.nebula.widget/src/org/polarsys/kitalpha/richtext/nebula/widget/MDERichTextConstants.java [hunk 3]
```diff
@@ -49,7 +49,8 @@
 	
 	public static final String MDE_SAVE_TOOLBAR = "mdesave"; 		//$NON-NLS-1$
 	public static final String MDE_SAVE = "MDESave";			//$NON-NLS-1$
-	
+	public static final String MDE_REFRESH = "Refresh"; //$NON-NLS-1$
```
Comment (bzendagui) [inline richtext/plugins/org.polarsys.kitalpha.richtext.nebula.widget/src/org/polarsys/kitalpha/richtext/nebula/widget/MDERichTextConstants.java:52]: Please update copyright year in the header

File: richtext/plugins/org.polarsys.kitalpha.richtext.widget.tools/src/org/polarsys/kitalpha/richtext/widget/tools/utils/Constants.java [hunk 4]
```diff
@@ -47,5 +47,6 @@
     public static final URL ADD_LINK_ICON = MDERichTextHelper.getURL(Activator.PLUGIN_ID, "icons/add_link.gif"); 	//$NON-NLS-1$
     public static final URL ADD_IMAGE_ICON = MDERichTextHelper.getURL(Activator.PLUGIN_ID, "icons/add_image.gif"); 	//$NON-NLS-1$
 	public static final URL SAVE_IMAGE_ICON = MDERichTextHelper.getURL(Activator.PLUGIN_ID, "icons/save_image.gif"); 	//$NON-NLS-1$
+	public static final URL REFRESH_IMAGE_ICON = MDERichTextHelper.getURL(Activator.PLUGIN_ID, "icons/refresh.gif");   //$NON-NLS-1$
```
Comment (bzendagui) [inline richtext/plugins/org.polarsys.kitalpha.richtext.widget.tools/src/org/polarsys/kitalpha/richtext/widget/tools/utils/Constants.java:50]: Please update copyright year in the header

File: richtext/plugins/org.polarsys.kitalpha.richtext.widget/src/org/polarsys/kitalpha/richtext/widget/factory/MDERichTextFactory.java [hunk 5]
```diff
@@ -34,6 +34,7 @@
 import org.polarsys.kitalpha.richtext.widget.tools.handlers.AddImageHandler;
 import org.polarsys.kitalpha.richtext.widget.tools.handlers.AddLinkHandler;
 import org.polarsys.kitalpha.richtext.widget.tools.handlers.ClearContentHandler;
+import org.polarsys.kitalpha.richtext.widget.tools.handlers.RefreshHandler;
```
Comment (bzendagui) [inline richtext/plugins/org.polarsys.kitalpha.richtext.widget/src/org/polarsys/kitalpha/richtext/widget/factory/MDERichTextFactory.java:37]: Please update copyright year in the header

File: richtext/plugins/org.polarsys.kitalpha.richtext.common/src/org/polarsys/kitalpha/richtext/common/impl/AbstractMDERichTextWidget.java (hunk 2 above)
Comment (arnauddieumegard) [reply richtext/plugins/org.polarsys.kitalpha.richtext.common/src/org/polarsys/kitalpha/richtext/common/impl/AbstractMDERichTextWidget.java:14]: Yes, the file previously had empty lines with tabs.
Corrected indentation replaces spaces with tabs.

**Review by bzendagui — APPROVED** [review]
Ok for me

**Pull request merged**